Classes/Psmb/Newsletter/Command/NewsletterCommandController.php

```diff
@@ -161,7 +161,9 @@ protected function createRequest() {
             $baseUri = new Uri($this->baseUri);
             $httpRequest->setBaseUri($baseUri);
         }
-        return new ActionRequest($httpRequest);
+        $actionRequest = new ActionRequest($httpRequest);
+        $actionRequest->setFormat('html');
```

Comment:
I already fixed this here https://github.com/psmb/Psmb.Newsletter/commit/a1bb6231fec1f0a0920f35c0512a3d133a1a6c00
The way you did it is a bit cleaner. If you want, you can rebase the change and remove my fix and leave yours.